modules/admin_manual/pages/configuration/files/big_file_upload_configuration.adoc

```diff
@@ -19,7 +19,7 @@
 
 * Make sure that the latest version of PHP, xref:installation/system_requirements.adoc[supported by ownCloud], is installed.
 * Disable user quotas, which makes them unlimited.
-* Your temp file or partition has to be big enough to hold multiple parallel uploads from multiple users; e.g. if the max upload size is 10GB and the average number of users uploading at the same time is 100: temp space has to hold at least 10 x 100GB.
+* Your temp file or partition has to be big enough to hold multiple parallel uploads from multiple users; e.g. if the average upload file size is *1GB* and the average number of users uploading at the same time is *100*: temp space has to hold at least *2 x 1 x 100 GB*. Twice as much space is required because the file chunks will be put together into a new file before it is finally moved into the user's folder.
```

Comment:
Wouldn't it be better to write "*2 x 100 GB*" instead of "*2 x 1 x 100 GB*"?

Reply:
Perhaps we should make an example where the numbers are clear, so nothing with 1, as it can be confusing if we take it out of the equation

Reply:
2 x 5GB average upload file size x 100 actively uploading Users

Reply:
Looks good.